tests/Microsoft.DotNet.UpgradeAssistant.TestHelpers/TargetFrameworkMonikerParser.cs

```diff
@@ -38,7 +38,7 @@ public static class TargetFrameworkMonikerParser
         public const string Net60_Android = "net6.0-android";
         public const string Net60_iOS = "net6.0-ios";
 
-        public const string Current = Net50;
+        public const string STS = Net50;
```

Comment:
As long as we're in here, we probably should update the tests to match STS/LTS/Preview TFM values (or maybe just change them to totally different "test" values). Not high priority, though.

Reply:
Agreed. I didn't want to do it in the same PR. I will take a follow-up item to fix up all the tests with the right values.

Reply:
Sounds good. Thanks!